.github/workflows/test.yaml

```diff
@@ -35,28 +35,28 @@ jobs:
         # We run this job multiple times with different parameterization
         # specified below, these parameters have no meaning on their own and
         # gain meaning on how job steps use them.
-        include:
-          - python: 3.5
-          - python: 3.6
-          - python: 3.7
-          - python: 3.8
-          - python: 3.9
+        jupyterlab_version: [1, 2, 3]
+        python: [3.6, 3.7, 3.8, 3.9]
 
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
           python-version: "${{ matrix.python }}"
-
+      - name: Build Python package
+        id: build-package
+        run: |
+          pip install jupyter_packaging wheel jupyterlab
+          python setup.py sdist bdist_wheel
```

Comment:
Test setup could be improved by caching the `./dist` directory. I would suggest moving that to a separate issue though. :)